pkg/azure/context.go

```diff
@@ -40,5 +40,12 @@ func NewAzContext(path string) (*AzContext, error) {
 		return nil, err
 	}
 
+	if context.VNetResourceGroup == "" {
```

Comment:
`context` is a bit misleading -- I thought this was Go's context, but it turns out it was the Azure context... wish we would have named this something different for readability (or name collision)

Reply:
sure! will do in a subsequent PR.